app/models/user_field.rb

```diff
@@ -15,10 +18,16 @@ class UserField < ActiveRecord::Base
 
   scope :public_fields, -> { where(show_on_profile: true).or(where(show_on_user_card: true)) }
 
+  enum :requirement, %i[optional for_all_users on_signup].freeze
```

Comment:
As this enum is backed by an integer column, we should probably follow [this rule from the Rails Style Guide](https://rails.rubystyle.guide/#enums):
> Prefer using the hash syntax for enum. Array makes the database values implicit & any insertion/removal/rearrangement of values in the middle will most probably lead to broken code.

Wdyt?

Reply:
I don't mind. If we want consistent enforcement we should enable the cop for this. 🙂

Reply:
Ah yes indeed :+1: I’ll open a PR.